app/oauth_test.go

```diff
@@ -69,9 +69,8 @@ func TestOAuthRevokeAccessToken(t *testing.T) {
 	th := Setup(t)
 	defer th.TearDown()
 
-	if err := th.App.RevokeAccessToken(model.NewRandomString(16)); err == nil {
-		t.Fatal("Should have failed bad token")
-	}
+	require.NotNil(t, th.App.RevokeAccessToken(model.NewRandomString(16)),
```

Comment:
i would make the call outside the check to make it clear, and also to make it similiar to the other test files. but im 0/5 on that